cardano-node/src/Cardano/Node/Protocol/Shelley.hs

```diff
@@ -163,14 +164,20 @@ readLeaderCredentials (Just ProtocolFilepaths {
     KesSigningKey kesKey <-
       firstExceptT FileError . newExceptT $ readFileTextEnvelope (AsSigningKey AsKesKey) kesFile
 
+    -- The label is used to distinguish multiple sets of credentials during
+    -- tracing. We use the public KES key as the label. TODO @deepfire works for
+    -- you? Or should we hash it first?
+    let label = T.pack (show (deriveVerKeyKES kesKey))
```

Comment:
@mrBliss, this looks fine, as we don't have a basis for preference yet.

Reply:
Same as for Byron, I'm curious what this looks like when traced.